include/itkPhaseCorrelationImageRegistrationMethod.h

```diff
@@ -187,6 +184,31 @@ class ITK_TEMPLATE_EXPORT PhaseCorrelationImageRegistrationMethod: public Proces
   itkGetConstObjectMacro( RealOptimizer,  RealOptimizerType );
   itkGetConstObjectMacro( ComplexOptimizer,  ComplexOptimizerType );
 
+  /** Given an image size, returns the smallest size
+   *  which factorizes using FFT's prime factors. */
+  SizeType RoundUpToFFTSize(SizeType inSize);
+
+  /** A size with all elements initialized to 0. */
+  const SizeType size0 = { 0 };
+
+  /** Set/Get the PadToSize.
+   *  Unset by setting a size of all zeroes.
+   *  size0 constant can be used for that purpose.
+   *
+   *  If PadToSize is set, image sizes are ignored and this size is used.
+   *
+   *  If used in a montage, a maximum image size can be determined,
+   *  RoundUpToFFTSize() called and the resulting size set as PadToSize. */
+  itkSetMacro(PadToSize, SizeType);
+  itkGetConstMacro(PadToSize, SizeType);
+
+  /** \class PaddingType
+   *  \brief Different kinds of padding the images to satisfy FFT size requirements.
+   *  \ingroup Montage */
+  enum class PaddingType { Zero, Mirror, MirrorWithExponentialDecay };
```

Comment:
PaddingType, PaddingKind -> better name would be PaddingMethod throughout

Reply:
Renamed!